app/experimenter/experiments/models/nimbus.py

```diff
@@ -283,21 +279,24 @@ def computed_enrollment_days(self):
             .first()
         )
         if paused_change:
-            return (paused_change.changed_on - self.start_date).days + 1
+            days = (paused_change.changed_on.date() - self.start_date).days
         else:
-            return self.proposed_enrollment
+            days = self.proposed_enrollment
+
+        # Always return at least 1
+        return max(days, 1)
```

Comment:
Wait what's this for?

Reply:
Going off the [test](https://github.com/mozilla/experimenter/blob/jh/5804/lets-date/app/experimenter/experiments/tests/test_models/test_models_nimbus.py#L371:L371), looks like we always want to return at least 1. I imagine this is so when you have an experiment that starts and ends on the same day, comparing the enrollment days would result in 0, which I guess looks a little odd maybe?

Reply:
No I would take this out.  The serializer validates that users can't enter less than 1, and if we want to set it to 0 for some other reason either through teh admin or through testing we should be able to.  I don't know any reason to have this and I can imagine it creating confusing behaviour in the future.

Reply:
Actually Just move it up to the paused_change exists case, and that should preserve the case the test is talking about and otherwise we just return the proposed_enrollment verbatim.  I think that's probably what we want.